puppet/services/neutron-honeycomb-agent.yaml

```diff
@@ -0,0 +1,84 @@
+heat_template_version: newton
+
+description: >
+  OpenStack Neutron Honeycomb agent configured with Puppet
+
+parameters:
+  ServiceNetMap:
+    default: {}
+    description: >
+      Mapping of service_name -> network name. Typically set via
+      parameter_defaults in the resource registry. This mapping overrides those
+      in ServiceNetMapDefaults.
+    type: json
+  DefaultPasswords:
+    default: {}
+    type: json
+  EndpointMap:
+    default: {}
+    description: Mapping of service endpoint -> protocol. Typically set
+                 via parameter_defaults in the resource registry.
+    type: json
+  HoneycombUser:
+    description: Username for Honeycomb agent
+    type: string
+    default: 'admin'
+  HoneycombPassword:
+    description: Password for Honeycomb agent
+    type: string
+    hidden: true
+    default: 'admin'
+  HoneycombRestPort:
+    description: Port for Honeycomb REST interface to listen on.
+    type: number
+    default: 8183
+  HoneycombNetconfSshPort:
```

Comment:
You use camelcase for Ssh here, but in opendaylight-api: OpendaylightHostconfURI:, URI is not Uri.  Good to be consistent here.  I see a mix of this in other services, I think 'SSH' should be fine for parameters.